README.md

```diff
@@ -1,12 +1,13 @@
 # ceph-dev-docker
 
-The purpose of this docker image is to help in the development of ceph.
+The purpose of this docker image is to help in the development of CEPH.
```

Comment:
Should be Ceph, instead of CEPH

Reply:
@Devp00l do you plan to rework this PR accordingly? It should be "Ceph"

Reply:
done